src/unicef_locations/forms.py

```diff
@@ -25,6 +25,7 @@ def clean(self):
         name_col = self.cleaned_data['name_col']
         pcode_col = self.cleaned_data['pcode_col']
         parent_code_col = self.cleaned_data['parent_code_col']
+        remap_table_name = self.cleaned_data['remap_table_name']
```

Comment:
what is remap table used for?

Reply:
comment please

Reply:
is it related to LocationRemapHistory?

Reply:
Remap table is the name of a CartoDB table which when is present it's used to remap old/outdated pcodes to new pcodes. This is related to location import, but it's used in LocationRemapHistory too.
`comment please`  You mean that i should add inline comments?